django/bosscore/migrations/0006_auto_20201002_2130.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2020-10-02 21:30
```

Comment:
Tim, can you say how you generated this migration file?

Reply:
`manage.py makemigrations --settings boss.settings.mysql bosscore`

I used `--settings boss.settings.mysql` because I did the locally.

Reply:
when i do this on the endpoint it names the migration file after the new tables added. How do you end up with the numbered migration file?

Reply:
Not sure. On your endpoint, you have the numbered migrations in `django/bosscore/migrations/`?

Reply:
I thought I replied to this. I'm pretty sure the output I got was the numbered migration file when I ran `makemigrations`.